### DISCUSSION TITLE
KF2EDG Connectors

### DISCUSSION THREAD

**Discussion by gsherwin74** [kachurovskiy/nanoels#83] [General]
Anyone have a good source in the US for the pluggable connectors.  5.08mm pitch is readily available but the 3.5mm are impossible to find.

**Comment by kachurovskiy** [comment]
Hi @gsherwin74, they should be readily available on AliExpress: https://www.aliexpress.com/item/1005002951702737.html

![image](https://user-images.githubusercontent.com/517919/202292463-eefd2740-d58f-4a7f-8e7a-197ba8657d60.png)

You can also find other offers with faster delivery - or delivery speeds up if you order something else as well to cross the ~15 dollar threshold.
